src/Microsoft.Build.Sql/sdk/Sdk.targets

```diff
@@ -55,7 +55,17 @@
         <!-- https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/BuildAssetsUtils.cs#L763 -->
         <PackageId>@(PackageReference->'%(Identity)'->Replace('.', '_'))</PackageId>
         <PackagePath>$(Pkg%(_ResolvedDacpacPackageReference.PackageId))</PackagePath>
-        <HintPath>%(_ResolvedDacpacPackageReference.PackagePath)/tools/%(Identity).dacpac</HintPath>
+        <IsSystemDacpac>@(PackageReference->'%(Identity)'->ToLowerInvariant()->Contains('microsoft.sqlserver.dacpacs'))</IsSystemDacpac>
+
+        <!-- File name of the dacpac inside the package -->
+        <DacpacName>%(PackageReference.DacpacName)</DacpacName>
+        <DacpacName Condition="'%(_ResolvedDacpacPackageReference.DacpacName)' == ''">%(PackageReference.Identity)</DacpacName>
+        <DacpacName Condition="'%(_ResolvedDacpacPackageReference.IsSystemDacpac)' == 'true'">master</DacpacName>
```

Comment:
does `DacpacName` get set to msdb when the user is trying to reference msdb?

Reply:
Yeah that's something users can set, we can test that during the bug bash